Formula/yelp-tools.rb

```diff
@@ -12,20 +12,45 @@ class YelpTools < Formula
     sha256 "66721baa2d6c6978cceac29f35e8112e96804b6ae0197d89175867f17caca2cd" => :mavericks
   end
 
+  depends_on "gettext" => :build
+  depends_on "gtk+3" => :run
+  depends_on "intltool" => :build
   depends_on "itstool" => :build
-  depends_on "pkg-config" => :build
-  depends_on "libxslt" => :build
   depends_on "libxml2" => :build
-  depends_on "yelp-xsl"
+  depends_on "libxslt" => :build
+  depends_on "pkg-config" => :build
+
+  resource "yelp-xsl" do
+    url "https://download.gnome.org/sources/yelp-xsl/3.18/yelp-xsl-3.18.0.tar.xz"
```

Comment:
`url "https://download.gnome.org/sources/yelp-xsl/3.18/yelp-xsl-#{yelp_tools_version}.tar.xz"` or similar to avoid these getting out of sync.